src/control/Control.Attribution.js

```diff
@@ -6,6 +6,8 @@ import * as DomEvent from '../dom/DomEvent';
 import * as DomUtil from '../dom/DomUtil';
 import Browser from '../core/Browser';
 
+const ukrainianFlag = '<svg xmlns="http://www.w3.org/2000/svg" width="12" height="8"><path fill="#4C7BE1" d="M0 0h12v4H0z"/><path fill="#FFD500" d="M0 4h12v3H0z"/><path fill="#E0BC00" d="M0 7h12v1H0z"/></svg>';

```

Comment:
Do we want to label this for a11y reasons?

e.g. using SVG `<title>` as a child of `<svg>`, or the global `title` attribute? Both these produce tooltips (only on hover in most browsers).

Otherwise SVG `<desc>` can be used, or global `aria-label`.

Reply:
Or else set to `<svg>` to `aria-hidden="true"`, as some screen readers may announce "Graphic" or similar.

Reply:
I think no, it's pure decoration — people following the link will get the meaning on the website anyway.